modules/table/src/main/java/org/apache/ignite/internal/table/distributed/index/MetaIndexStatus.java

```diff
@@ -70,6 +71,9 @@ public enum MetaIndexStatus {
     private static final MetaIndexStatus[] VALUES_INDEXED_BY_CODE;
 
     static {
+        long distinctCodes = Arrays.stream(values()).map(s -> s.code).distinct().count();
```

Comment:
Formally, method 123 creates a new array each time, which is a bit suboptimal.

Reply:
`values()` is called 4 times in this static initialization block. This creates 4 tiny arrays, which is absolutely not a problem for a one-off action. Introducing a variable will harm the understanding of the code more